src/System.Runtime.Loader/tests/DefaultContext/DefaultLoadContextTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace System.Runtime.Loader.Tests
+{
+    public class DefaultLoadContextTests
+    {
+        private static Assembly ResolveAssembly(AssemblyLoadContext sender, AssemblyName assembly)
+        {
+            string assemblyFilename = assembly.Name + ".dll";
+            
+            return sender.LoadFromAssemblyPath(Path.Combine(Path.GetTempPath(), assemblyFilename));
+        }
+
+        private static void Init()
+        {
+            // Delete the assembly from the temp location if it exists.
+            var assemblyFilename = "System.Runtime.Loader.Noop.Assembly.dll";
+            var targetPath = Path.Combine(Path.GetTempPath(), assemblyFilename);
+            if (File.Exists(targetPath))
+            {
+                File.Delete(targetPath);
+            }
+
+            // Rename the file local to the test folder.
+            var sourcePath = Path.Combine(Directory.GetCurrentDirectory(),assemblyFilename);
+            var targetRenamedPath = Path.Combine(Directory.GetCurrentDirectory(), "System.Runtime.Loader.Noop.Assembly_test.dll");
+            if (File.Exists(sourcePath))
+            {
+                if (File.Exists(targetRenamedPath))
+                {
+                    File.Delete(targetRenamedPath);
+                }
+
+                File.Move(sourcePath, targetRenamedPath);
+            }
+
+            // Finally, copy the file to the temp location from where we expect to load it
+            File.Copy(targetRenamedPath, targetPath); 
+        }
+
+        [Fact]
+        public static void LoadInDefaultContext()
+        {
+            Init();
+
+           // This will attempt to load an assembly, by path, in the Default Load context via the Resolving event
+           var assemblyName = "System.Runtime.Loader.Noop.Assembly";
+
+            AssemblyLoadContext.Default.Resolving += ResolveAssembly;
+            var assemblyExpected = AssemblyLoadContext.Default.LoadFromAssemblyName(new AssemblyName(assemblyName));
+            
+            // We should have successfully loaded the assembly in default context.
+            Assert.NotNull(assemblyExpected);
+
+            // And make sure the simplename matches
+            Assert.Equal(assemblyExpected.GetName().Name, assemblyName);
```

Comment:
Nit: the arguments should be switched, as the first arg to Equal is ```expected``` and the second is ```actual```